Sources/Relays/CurrentValueRelay.swift

```diff
@@ -75,14 +75,15 @@ private extension CurrentValueRelay {
         func forceFinish() {
             self.sink?.shouldForwardCompletion = true
             self.sink?.receive(completion: .finished)
+            self.sink = nil
         }
```

Comment:
Since now `cancel` and `forceFinish` are the same you can simplify and maybe just keep the `cancel` method.

Reply:
I think the intent of the method is clearer by leaving it in the `forceFinish` method. It is called by `cancel` method of `CurrentValueRelay.Subscription` and `deinit` method of `CurrentValueRelay.`

The method isn't cancelling the subscription, it is finishing the subscription. It just happens to be called from `cancel`, because of how a `CurrentValueRelay` works.